### DISCUSSION TITLE
Integration of Sonos?

### DISCUSSION THREAD

**Discussion by TheEsch** [MiczFlor/RPi-Jukebox-RFID#1211] [General]
Hello,

i´m new to that project. 
Andi i am very interested.
Is there a possibility to start a playlist for the sonos Applikaton via RFID?
I´m asking, because my daughters each have a sonos speaker in their bdroom. 
And it would be great if they could start  their  own playlists via RFID.
Does this possibility exist?

**Comment by s-martin** [comment]
I don’t know, if Sonos has an API you could use?

**Comment by tilpm** [comment]
There is a Sonos API - https://github.com/jishi/node-sonos-http-api
Anyone got this to work?
